fv3core/stencils/basic_operations.py

```diff
@@ -20,21 +20,30 @@ def copy_stencil(q_in: sd, q_out: sd):
         q_out = q_in
 
 
-def copy(q_in, origin=(0, 0, 0), domain=None):
+def copy(q_in, origin=(0, 0, 0), domain=None, use_cache=True):
```

Comment:
hopefully at some point soon we don't need this function at all, and all copies are done in another stencil

Reply:
Yes, since it's a two-line function we could replace it already pretty easily, wherever it is called. I think it would be fine to do that whenever, to move all the calls to `make_storage_from_shape` to the same place (the compute functions).